huey_monitor/tqdm.py

```diff
@@ -21,7 +21,8 @@ def __init__(self,
                  total=None,
                  unit='it',
                  unit_divisor=1000,
-                 parent_task_id=None
+                 parent_task_id=None,
+                 cumulate_w_parent_progress=True,
```

Comment:
Can you rename `cumulate_w_parent_progress` to something shorter? e.g.: `cumulate2parents` ?